docs/building/windows-instructions.md

```diff
@@ -69,7 +69,7 @@ After successfully building, run tests in the Visual Studio Test Explorer window
 
 ### Running tests from the command line
 
-From the root, run `build.cmd` and then `build.cmd -runTests`.
+From the root, run `build.cmd` and then `build.cmd -test`.
 For more details, or to test an individual project, you can navigate to the test project directory and then use `dotnet test`
 
 ## Known Issues
```

Comment:
Are the instructions for running benchmarks changed? https://github.com/dotnet/machinelearning/blob/c2f8ed5c9c478194fa6c7e387bc9f8fc43e4e0bc/test/Microsoft.ML.PerformanceTests/README.md

Also can you add a link to bechmarking from the build page? I'm hoping to get the community more involved with the speed as it's both a highlight of ML․NET and something to work on.
```suggestion
## Running Benchmarks
See: [benchmarking instructions](../../test/Microsoft.ML.PerformanceTests/README.md)
## Known Issues
```